app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -86,9 +88,15 @@ class ProfileFragment : Fragment() {
 
                 Picasso.get()
                         .load(it.avatarUrl)
-                        .placeholder(requireDrawable(requireContext(), R.drawable.ic_person_black_24dp))
+                        .placeholder(requireDrawable(requireContext(), R.drawable.ic_account_circle_grey_24dp))
                         .transform(CircleTransform())
                         .into(rootView.avatar)
+
+                val imageButton: ImageButton = rootView.edit_profile_button
+                imageButton?.setOnClickListener {
```

Comment:
`rootView.edit_profile_button.setOnClickListener` should be written as

Reply:
> `rootView.edit_profile_button.setOnClickListener` should be written as

@nikit19 You are right! I corrected it.